BookingSystem/BookingSystem.Commands/Commands/HotelCommands/Commands/EditHotelCommand.cs

```diff
@@ -28,17 +28,18 @@ public EditHotelCommandHandler(BookingWriteContext dataContext, IMapper mapper)
             _mapper = mapper;
         }
 
-        public async Task<Result> Execute(EditHotelCommand command)
+        public async Task<Result> ExecuteAsync(EditHotelCommand command)
         {
-            var dto = command.Hotel;
+            var hotelDto = command.Hotel;
 
-            var hotel = await _dataContext.Hotels.FindAsync(dto.HotelId);
+            var hotel = await _dataContext.Hotels.FindAsync(hotelDto.HotelId);
             if (hotel == null)
```

Comment:
Why do you use isActive for hotel, but isAvailable for extra services. Are there different delete operations?

Reply:
Fixed.